models/src/main/java/com/vimeo/networking2/Subscriptions.kt

```diff
@@ -0,0 +1,56 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * A model that holds the type of push subscriptions a user has. This class
+ * has both getters and setters since a user can update their subscriptions
```

Comment:
docs on this class are incorrect as they references setters which this data class does not have.